main.py

```diff
@@ -36,7 +38,14 @@
     }
 
     model = make_simple_cnn()
-    img_path = "../images/train/true/IMG_6605.jpg"
     train(model, params)
-    save_model(model, "dupa")
-    plot_heatmaps(img_path, params['img_size'], model, "dupa.jpg")
+    save_model(model, "first_working_model")
```

Comment:
We have callback for saving best model, so explicit save isn't necessary (what's more it may be worse, because it saves last state of model, not best one).

If there is a reason for saving model directly use `args.name` for its name.

Reply:
Done